app/data/events.json

```diff
@@ -1,4 +1,40 @@
 [
+	{
+		"date": "2024-09-29T18:00:00.000-04:00",
+		"link": "https://www.eventbrite.com/e/boston-ts-club-vi-rise-of-the-squiggles-tickets-1014606444187",
+		"location": "Microsoft NERD Center",
+		"time": "5:30pm - 8:30pm",
+		"topics": [
+			{
+				"author": {
+					"name": "Josh Goldberg",
+					"url": "https://joshuakgoldberg.com/"
+				},
+				"title": "What's New in TypeScript 5.5"
+			},
+			{
+				"author": {
+					"name": "Paula Paul",
+					"url": ""
```

Comment:
@JoshuaKGoldberg do you happen to have URLs for these speakers?

Reply:
https://x.com/paulapaultweets?lang=en

https://x.com/corinamurg?lang=en

https://blackgirlbytes.dev/

Reply:
Amazing, thank you!